### ISSUE TITLE
Is the documentation "Expand parameters" supposed to be JSDoc?

### ISSUE THREAD

**Issue by kee-oth** [ramda/ramda#3240]
Hi!

In the Ramda docs, clicking on "Expand parameters" gives you a sort of JSDoc style of documentation but it looks incomplete.

`add`, for example, only shows parameter names but not their types. You can see in the HTML on the page that the `span.type` element is empty.

 https://ramdajs.com/docs/#add
```
a
b
Returns Number
```

The HTML
```
<li class="list-group-item">
  <span class="type"></span>
  <span class="name">a</span>
  <span class="description"></span>
</li>
```

Is this intentional? If it is, is the intention that developers would look at the Hindley-Milner representation and the JSDoc style representation and put them together in their head to get a full picture?

**Comment by CrossEye** [comment]
No, I think that's a bug.  I'm pretty sure that used to work.  The descriptions from the source code comment block as well as the names are showing up.  (There just happen to be no descriptions for `add`'s parameters.)  But the type should also show.  I'm not sure when this broke, but we should fix it.  I don't know when it broke, as I never look at those, preferring the HM-annotations.  But we'll try to get to it soon.

**Comment by kee-oth** [comment]
Gotcha, thanks so much for the response!

**Comment by kee-oth** [comment]
Commenting to re-open from accidental closing 🤦‍♂️
